src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.cs

```diff
@@ -4316,7 +4317,7 @@ public static bool TryWrite(this Span<char> destination, IFormatProvider? provid
 
             /// <summary>Gets the current element of the enumeration.</summary>
             /// <returns>Returns a <see cref="Range"/> instance that indicates the bounds of the current element withing the source span.</returns>
-            public Range Current => new Range(_startCurrent, _endCurrent);
+            public readonly Range Current => new Range(_startCurrent, _endCurrent);
```

Comment:
This is not specified in API review. Given the typical usage of ref struct, this won't really have impact.